StoryMap/StoryMap/Screens/StoryDetail/Coordinator/StoryDetailCoordinator.swift

```diff
@@ -5,30 +5,36 @@
 //  Created by Dory on 02/11/2021.
 //
 
-import Foundation
 import UIKit
+import Combine
 
 class StoryDetailCoordinator: CoordinatorType {
-    var presenter = UINavigationController()
-    var onDidStop: (() -> Void)?
-	var onDeleteStory: ((Story) -> Void)?
+	var presenter: UINavigationController
+	var story: Story
+	
+	var deleteStorySubject = PassthroughSubject<Story, Never>()
```

Comment:
I suspect the coordinator should not hold a PassthroughSubject.
These should be in ViewModel. This on is not clear for me, if any at all is subscribing to it.